docs/connecting-data/connecting-to-apis-and-external-data/servicenow-api.md

```diff
@@ -1,7 +1,150 @@
----
-description: How to Connect DataDistillr to the ServiceNow API
----
-
 # ServiceNow API
 
-Details coming soon!
\ No newline at end of file
+## Creating a ServiceNow account
+Set up an account with [ServiceNow](https://www.servicenow.com/).
+
```

Comment:
Missing cost section

Reply:
That's fine, we still need a section to say this and link to where they can find cost information